src/stavemodifier.ts

```diff
@@ -0,0 +1,152 @@
+// [VexFlow](http://vexflow.com) - Copyright (c) Mohit Muthanna 2010.
+//
+// ## Description
+// A base class for stave modifiers (e.g. clefs, key signatures)
+
+import { Element } from './element';
+import { Glyph } from './glyph';
+import { Stave } from './stave';
+
+export interface LayoutMetrics {
+  xMin: number;
+  xMax: number;
+  paddingLeft: number;
+  paddingRight: number;
+}
+
+export interface Metrics {
```

Comment:
I don't think we need this -- this looks like the metrics we use for `StaveNote`